IOSExercises/Views/ColorsTableViewController.m

```diff
@@ -0,0 +1,64 @@
+//
+//  ColorsTableViewController.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/27/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "ColorsTableViewController.h"
+#import "EXRColor.h"
+
+static NSString *kCellIdentifier = @"colorCell";
```

Comment:
Any issues with using `const`? (Sometimes the compiler complains, but in case it doesn't, please include `const` descriptor)